index.js

```diff
@@ -14,6 +14,10 @@ function defaultKey (req, file, cb) {
   })
 }
 
+function defaultAcl (req, file, cb) {
+  setImmediate(function () { cb(null, 'private') })
```

Comment:
I realise now that the other call looks like this as well, so feel free to leave it and then I'll submit a PR to fix both of them later...